dll/win32/srclient/srclient_main.c

```diff
@@ -29,40 +29,33 @@
 
 BOOL
 WINAPI
-SRSetRestorePointA(PRESTOREPOINTINFOA pRestorePtSpec, PSTATEMGRSTATUS pStateMgrStatus)
+SRSetRestorePointA(PRESTOREPOINTINFOA pRestorePtSpec, PSTATEMGRSTATUS pSMgrStatus)
 {
-    RESTOREPOINTINFOW RPInfoW;
-
     DPRINT1("SRSetRestorePointA is unimplemented\n");
 
-    if (!pRestorePtSpec)
+    if (!pSMgrStatus)
         return FALSE;
 
-    RPInfoW.dwEventType = pRestorePtSpec->dwEventType;
-    RPInfoW.dwRestorePtType = pRestorePtSpec->dwRestorePtType;
-    RPInfoW.llSequenceNumber = pRestorePtSpec->llSequenceNumber;
-
-    if (!MultiByteToWideChar( CP_ACP, 0, pRestorePtSpec->szDescription, -1, RPInfoW.szDescription, MAX_DESC_W))
-        return FALSE;
+    pSMgrStatus->nStatus = ERROR_SUCCESS;
+    pSMgrStatus->llSequenceNumber = pRestorePtSpec->llSequenceNumber + 1;
 
-    return SRSetRestorePointW(&RPInfoW, pStateMgrStatus);
+    return TRUE;
 }
 
 
 BOOL
 WINAPI
-SRSetRestorePointW(PRESTOREPOINTINFOW pRestorePtSpec, PSTATEMGRSTATUS pStateMgrStatus)
+SRSetRestorePointW(PRESTOREPOINTINFOW pRestorePtSpec, PSTATEMGRSTATUS pSMgrStatus)
 {
-
     DPRINT1("SRSetRestorePointW is unimplemented\n");
 
-    if (!pStateMgrStatus)
+    if (!pSMgrStatus)
         return FALSE;
 
-    memset(pStateMgrStatus, 0, sizeof(STATEMGRSTATUS));
```

Comment:
This seems arbitrary. Is there a test that shows that the whole structure _isn't_ zeroed?